src/ResourceManager/ManagementGroups/Commands.ManagementGroups/Cmdlets/UpdateAzureRmManagementGroup.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Management.Automation;
+using Microsoft.Azure.Commands.ManagementGroups.Common;
+using Microsoft.Azure.Commands.ManagementGroups.Models;
+using Microsoft.Azure.Management.ResourceManager;
+using Microsoft.Azure.Management.ResourceManager.Models;
+
+namespace Microsoft.Azure.Commands.ManagementGroups.Cmdlets
+{
+    [Cmdlet("Update", "AzureRmManagementGroup", DefaultParameterSetName = Constants.ParameterSetNames.GroupOperationsParameterSet, SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.Medium), OutputType(typeof(string))]
```

Comment:
@rajshah11 same comment about updating the output type of this cmdlet to reflect what is written to the output stream below

Reply:
Done. Thanks!